scania-digital-design-system/tegel
# build: update to node 18

**Describe pull-request**  
Updating code to use version 18

**Solving issue**  
Fixes: DTS-1706

**How to test**  
1. Open this branch on your local machine
2. Try to run it on your current version of node - anything below node version 18.16 should result in error
3. Install node 18
4. Run the code again (`npm run build` and `npm run start`)
5. If everything runs OK, we are good to go
6. Check below in case you are Mac or Windows user. It would be nice to have confirmation it works on both OS.

Important to check:

- [x] Code was tested on Mac
- [x] Code was tested on Windows


**Additional context**  
I have put `postcss: false` as that feature was creating an issue for us on node 18. There is a possibility for a workaround, but I am not sure we need it as we load a CSS file (tegel.css) which is already compiled by Stencil. In version 7 of Storybook, this feature is anyway deprecated. If someone thinks this should be enabled by workarounds, please write in the comments.
